api/src/main/java/org/pentaho/metaverse/api/model/OCIResourceInfo.java

```diff
@@ -0,0 +1,127 @@
+/*! ******************************************************************************
```

Comment:
So I have a couple of issues with this approach:
1- Why not have subclassesed JDBCResourceInfo - I don't see why you would copy/paste all that other stuff into a new class.
2- The data/indeces database information is relevant to Oracle in general, not just for Oracle using the OCI connector. So if it's imperative to capture that tablespace information for Oracie OCI, it's equally important to capture it for Oracle using pure JDBC. And, given that other existing (and future) database connections may have data that needs captured as well (like the Web App Name for Pentaho Data Services for example), it seems like the wrong place to be doing this. I personally think it would be better to add a getter to a DatabaseMeta that would return a JDBC/JNDI resource info object that could be consumed by the MetaVerse.

Reply:
My comments to your points:
1 - I agree with OCIResourceInfo being a subclass of JdbcResourceInfo, avoiding repetition of some methods and fields is always a code idea, my bad for not seeing it before.
2 - I don't fully understand your idea here. I mean, JDBC connector is no exclusive from Oracle, it might be used to other db connections. Additionally, DatabaseMeta class is from Kettle Core project, it might be riskier to perform changes on that class.

Should I proceed with making OCIResourceInfo a subclass of JDBCResourceInfo? Is there any additional changes that I should add also? Ty

Reply:
@ricardosilva88 move fwd with the subclass and get this in, please. @pentaho-lmartins